lib/cog_cmd/format/table.rb

```diff
@@ -36,4 +38,26 @@ def run_command
     response.template = 'preformatted'
     response['body'] = table.to_s
   end
+
+  def mappings(args)
+    args.reduce([]) do |acc, raw_arg|
+      name, selector = if raw_arg.include?("=")
+                         raw_arg.split("=", 2)
+                       elsif raw_arg.include?(".")
+                         [raw_arg.split(".").last, raw_arg]
+                       else
+                         [raw_arg, raw_arg]
+                       end
+
+      name = if acc.find{|m| m[:name] == name}
+               raw_arg # name was already taken
+             else
+               name
+             end
+
+      acc << {name: name,
+              path: JsonPath.new("$.#{selector}")}
+    end
+  end
```

Comment:
Do we support grabbing a key with a dot or equals sign in it? If not, I guess that's ok, but it might be worth calling out in the docs.